src/css.js

```diff
@@ -113,6 +113,13 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableDimensionsInFullscreenMode && window.top !== window.self &&
+		document.msFullscreenElement ) {
+		val = Math.round( elem.getBoundingClientRect()[ name ] * 100 );
```

Comment:
Uhm, no - just multiplying `offsetWidth` by 100 is incorrect and this function *is* returning integer values, Gentlemen :-)

Look - if the clientWidth is `314` and the IE 11 bug happens, we'll get the following values:

```javascript
elem.offsetWidth == 3
elem.getBoundingClientRect().width == 3.14
Math.round( elem.getBoundingClientRect().width * 100 ) == 314
elem.offsetWidth * 100 == 300
```
which makes this value still integer but actually correct (`314` as expected versus `300` which your suggestion would return)